apps/meteor/ee/client/omnichannel/tags/TagEditWithData.js

```diff
@@ -1,23 +1,22 @@
 import { Callout } from '@rocket.chat/fuselage';
-import { useTranslation } from '@rocket.chat/ui-contexts';
+import { useEndpoint, useTranslation } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
 import React from 'react';
 
 import { FormSkeleton } from '../../../../client/components/Skeleton';
-import { AsyncStatePhase } from '../../../../client/hooks/useAsyncState';
-import { useEndpointData } from '../../../../client/hooks/useEndpointData';
 import TagEdit from './TagEdit';
 import TagEditWithDepartmentData from './TagEditWithDepartmentData';
 
 function TagEditWithData({ tagId, reload, title }) {
-	const { value: data, phase: state, error } = useEndpointData('/v1/livechat/tags/:tagId', { keys: { tagId } });
-
+	const getTag = useEndpoint('GET', '/v1/livechat/tags/:tagId', { tagId });
+	const { data, isLoading, isError } = useQuery(['/v1/livechat/tags/:tagId', tagId], () => getTag(), { enabled: Boolean(tagId) });
```

Comment:
Why in the other one you had `!!value` and here `Boolean(value)`? Should we keep consistency?

Reply:
You are completely right!! I was doing some tests and I ended up forgetting to make things consistent. Thanks, Kevin